packages/create-hintrc/package.json

```diff
@@ -10,8 +10,10 @@
     "create-hintrc": "./dist/src/create-hintrc.js"
   },
   "dependencies": {
+    "@hint/utils": "^1.0.0",
```

Comment:
All of these should be `devDependencies`. We bundle everything with webpack and otherwise they will get pulled when running `npm create`